### PR TITLE
Add changelog for latest releases

### PR DESCRIPTION
**Pull request by laurakalbag**
Preview: https://docs-2d4p37fmh-statelyai.vercel.app/blog/2023-09-21-context-schema-and-export-to-external-editors

### CODE REVIEW

File: blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+title: Context schema and export to CodeSandbox or StackBlitz
+description: You can now open your machine in CodeSandbox or StackBlitz. These options show when you select either JavaScript or TypeScript export formats in the Code panel.
+authors: [laura]
+tags: [changelog, new, improved, fixed]
+date: 2023-09-21
+slug: 2023-09-21-context-schema-and-export-to-external-editors
+image: /blog/2023-09-21-context-schema-and-export-to-external-editors.png
+---
+
+You can now open your machine in CodeSandbox or StackBlitz. These options show when you select either JavaScript or TypeScript export formats in the **Code** panel. When you open your machine in these external editors, it will be deployed as a demo app so you can jump quickly into prototyping and get an example of how to integrate your machine into your codebase.
```
Comment (kevinmaes) [inline blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx:11]: Nit: The order of the features doesn't match the order in the title. Should the opening paragraph about exports get its own header and then be moved under the Context schema section?

File: blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+![The Context schema section at the end of the Machine details panel. The schema contains properties for timers and volume.](./context-schema.png)
+
+You can edit any context properties in the modal that opens when you click the property in the schema list and delete all of them from the **Delete all** option under the **...** context menu.
+In the Context schema panel, you can also search through the list of context properties in your machine by name.
+
+## Improved Simulation mode
+
+We’ve made some improvements to Simulation mode. The log panel is now named the **Controls** and not only shows a log of your simulated path but contains buttons to navigate through the currently available events easily.
```
Comment (kevinmaes) [inline blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx:28]:
```suggestion
We’ve made some improvements to Simulation mode. The log panel is now named the **Simulation controls** and not only shows a log of your simulated path but contains buttons to navigate through the currently available events easily.
```

The screenshot shows the panel name as "Simulation controls".

File: blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+<!--truncate-->
+
+## Context schema
+
+We’ve now released the context schema in the editor. You can find the context schema above the event schemas in the machine **Details** panel. Here, you can add a new context property with its name, type (string, number, boolean, array and object), and an optional description. These will then be exported in your TypeScript code. When you import a machine, Stately will stub an empty schema for any context properties in your code. If you add a context property in the **Context** panel that’s missing from the context schema, a quick link will appear under that property to quickly add it to the schema.
+
+![The Context schema section at the end of the Machine details panel. The schema contains properties for timers and volume.](./context-schema.png)
+
+You can edit any context properties in the modal that opens when you click the property in the schema list and delete all of them from the **Delete all** option under the **...** context menu.
```
Comment (kevinmaes) [inline blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx:23]:
```suggestion
You can edit any context properties in the modal that opens when you click the property in the schema list. To delete all schema properties, use the **Delete all** option under the **...** menu.
```

I'm suggesting to split editing and delete all into two separate sentences.
You could keep the "**...** context menu" but I found that in this context, that "context" is confused with the **Context** feature being described.

File: blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+In the Context schema panel, you can also search through the list of context properties in your machine by name.
```
Comment (kevinmaes) [inline blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx:24]:
```suggestion
In the Context schema section of the Details panel, you can also search through the list of context properties in your machine by name.
```

The panel itself was previously referred to as the "Details panel". There's already its own Context panel so I thought adding "Context schema panel" was a bit confusing.

**Review by kevinmaes — CHANGES_REQUESTED** [review]
A couple of minor issues. Looks good overall though.

File: blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx (hunk 1 above)
Comment (laurakalbag) [reply blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx:11]: The order in the heading is just for readability. Export formats are best to be first as I think they’ll be more popular with more users for now.

Comment (kevinmaes) [reply blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx:11]: Understood. Thanks!

**Review by kevinmaes — APPROVED** [review]
Great!

**Pull request merged**